### DISCUSSION TITLE
MWE for numbered equations in a .docx document

### DISCUSSION THREAD

**Discussion by murattasdemir** [quarto-dev/quarto-cli#8137] [Q&A] [answered]
### Description

This might be a basic question but I couldn't find a straight answer neither in Q&A not in issues. I cannot get equations numbered in quarto docx document. What is the simplest way of doing it? Better if the numbers are right aligned.

**Comment by cderv** [comment]
What did you try so far ? 

Equations syntax is documented at: https://quarto.org/docs/authoring/cross-references.html#equations

![image](https://github.com/quarto-dev/quarto-cli/assets/6791940/6bd1d8a9-66a0-46f9-939d-86d1ea98e0df)

**Comment by murattasdemir** [comment]
The following
```
---
title: "MWE"
format: docx
---
$$
    \begin{equation}x_{i}(p_1,p_2, w)=h_{i}(p_1,p_2, \bar{u})\end{equation}
$${#eq:eqlt}

$$x_{i}(p_1,p_2, e(p_1,p_2, \bar{u}))=h_{i}(p_1,p_2, \bar{u})$${#eq:eq1}
```
gives me this
<img width="934" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/40790433/02df37e3-904a-4dcf-a2b2-dd99540c7668">

**Reply by mcanouil** [reply, marked as answer]
You are not using the syntax from the documentation.
It's `eq-` not `eq:`.
This is written in the first line at <https://quarto.org/docs/authoring/cross-references.html#equations>.

**Reply by murattasdemir** [reply]
I knew it would be such a basic mistake . Thanks a lot. 😔
